DocGen4/Output/Base.lean

```diff
@@ -55,12 +98,19 @@ abbrev BaseHtmlM := BaseHtmlT Id
 abbrev HtmlT (m) := ReaderT SiteContext (BaseHtmlT m)
 abbrev HtmlM := HtmlT Id
 
+abbrev ModuleToHtmlT (m) := StateT ModuleToHtmlContext (HtmlT m)
```

Comment:
I don't think there is a need to introduce an additional monad here, just adding the state transformer on top of `HtmlT` directly should be fine.

Reply:
Do you mean adding `backrefs` to `SiteContext` and `abbrev HtmlT (m) := StateT SiteContext (BaseHtmlT m)` instead of `ReaderT`?

Reply:
`abbrev HtmlT (m) := StateT YourThing <| ReaderT SiteCcontext (BaseHtmlT m)

Reply:
Done.